web/pages/app.module.css

```diff
@@ -14,14 +14,38 @@
     position: sticky;
     top: 0;
     left: 0;
+    transform: translateY(-100%);
+  }
+
+  .slideOut {
+    animation: slideOut 0.5s cubic-bezier(0.25, 0.92, 0.37, 1) forwards;
```

Comment:
Could all of this be simplified if we use `transition` instead of `animation`? Something like (not tested)

```
.header.onFrontPage {
  position: fixed;
  transform: translateY(-100%);
  transition: translateY 0.5s cubic-bezier(0.25, 0.92, 0.37, 1);
}

.header.onScrolledFrontPage {
  transform: translateY(0%);
}
```

This way I'm thinking there's no need for multiple keyframes/animations, `seenHeader` or `slideOut` class.

Reply:
Another great suggestion! bc6daef 

(~Note that this re-introduces the issue of the header not appearing of the page is loaded with a non-zero scrollTop position.~ Also fixed in 2fd8be9.)